src/common/obj_bencher.cc

```diff
@@ -376,7 +376,17 @@ int ObjBencher::fetch_bench_metadata(const std::string& metadata_file,
   } else {
     *op_size = *object_size;
   }
-  unsigned ops_per_object = *object_size / *op_size;
+
```

Comment:
the title of the commit message

> Fix rand & seq read bench to use num ops instead of num …
…objs

 is not specific and does not summarize the change in this commit

Reply:
Okay, let's agree on a commit msg before I do the changes so I could do everything in one shot. From my perspective - these changes boil down to fixing where we incorrectly used object count in place of ops count. For example the most relevant changes were replacing `num_objects > data.started` with `num_ops > data.started` in seq read and replacing `rand_id = rand() % num_objects` with `rand_id = rand() % num_ops` in rand read.

How about this: "tools/rados: Fix rand & seq read bench to use num ops instead of num objs when issuing and tracking outstanding IO"? I am also open to other suggestions :)

Reply:
@tchaikov thoughts?

Reply:
Since there is no objection, I've gone ahead and updated to "tools/rados: Fix rand & seq read bench to use num ops instead of num objs when issuing and tracking outstanding IO"

Reply:
@tchaikov please let me know if you are good with the changes. Thanks!

Reply:
sorry for the latency. the commits look good. i edited the title of the PR to fit it in 72 chars.

Reply:
No worries. Thanks for the edit @tchaikov !